cmd/provider/main.go

```diff
@@ -108,6 +110,16 @@ func main() {
 	if *enableExternalSecretStores {
 		o.SecretStoreConfigGVK = &v1alpha1.StoreConfigGroupVersionKind
 		log.Info("Alpha feature enabled", "flag", features.EnableAlphaExternalSecretStores)
+
+		o.ESSOptions = &tjcontroller.ESSOptions{}
+		if *essTLSCertsPath != "" {
```

Comment:
My understanding is that it's okay to enable the ESS feature (via the `--enable-external-secret-stores` command-line option) but not to configure the TLS certificate path (i.e., not giving the `--ess-tls-cert-dir` option). How would be behave in this case?

Reply:
It will fallback to the existing in-tree Vault implementation.